app/src/main/java/com/android/streetworkapp/ui/tutorial/TutorialSignIn.kt

```diff
@@ -0,0 +1,174 @@
+package com.android.streetworkapp.ui.tutorial
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.DirectionsRun
+import androidx.compose.material.icons.filled.ConnectWithoutContact
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.android.sample.R
+import com.android.streetworkapp.ui.authentication.IconAndTextRow
+
+@Composable
+fun TutorialSignIn() {
+  // Set number of page
+  val pagerState = rememberPagerState(pageCount = { 3 })
```

Comment:
Question: Isn't it dangerous to hardcode the size of the tutorial if you add a page that breaks everything? Isn't there a way to put a list of screens, then just take the number of elements in the list?

Reply:
https://developer.android.com/develop/ui/compose/layouts/pager?hl=fr
Dans les exemple que je vois il sont tous hardcoder

Reply:
Ok seems legit to me resolved